themes/default/content/templates/container-service/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Container Templates
```

Comment:
I notice you called these Container Templates but the AWS one is called "Container Service Templates" we should settle on a name here

Reply:
Suggest “container service templates” over “container templates”.